editor/src/components/canvas/canvas-strategies/strategies/drag-to-move-metastrategy.spec.browser2.tsx

```diff
@@ -12,7 +12,7 @@ const TestProject1 = `
     data-testid='child-1'
   />
   <div
-    style={{ backgroundColor: '#aaaaaa33', position: 'absolute', left: 100, top: 150, width: 200, height: 100 }}
```

Comment:
why this change in the test?

Reply:
This test was used to check that the `DO_NOTHING` strategy would be applicable, and it no longer was as a result of this PR.